### ISSUE TITLE
Installed executable does nothing and silently exits

### ISSUE THREAD

**Issue by srackham** [srackham/rimu#3]
If installed from Github the executable does nothing and returns immediately.

But bizarrely, if it is installed from deno.land it works:

```
$ deno install -A -f --name rimudeno https://deno.land/x/rimu@11.1.12/src/deno/rimuc.ts
✅ Successfully installed rimudeno
/home/srackham/.deno/bin/rimudeno

$ /home/srackham/.deno/bin/rimudeno --version
11.1.12
```

Installing from Github the bundled executable command does nothing, it simply returns immediately:

```
$ deno install -f -A --name rimudeno https://raw.githubusercontent.com/srackham/rimu/11.1.12/src/deno/rimuc.ts
✅ Successfully installed rimudeno
/home/srackham/.deno/bin/rimudeno

$ /home/srackham/.deno/bin/rimudeno
        <------ Blank line instead of version number
$
```

I suspect this behavior is linked to a Deno 1.4.0 install command regression: https://github.com/denoland/deno/issues/7455

**Comment by srackham** [comment]
The same behaviour is exhibited by eggs 2.2 on Deno 1.4.0 mentioned in https://github.com/nestdotland/eggs/issues/73

Maybe there something about the Deno 1.4.0 `install` command bundling that is deno.land specific (`eggs` is also fetched from a non deno.land registry URL).

**Comment by srackham** [comment]
Fixed in Deno 1.4.1
